ssl/ssl_sess.c

```diff
@@ -828,6 +828,13 @@ long SSL_SESSION_set_time(SSL_SESSION *s, long t)
     return (t);
 }
 
+int SSL_SESSION_get_protocol_version(const SSL_SESSION *s)
+{
+    if (s == NULL)
+        return (0);
```

Comment:
Is this necessary. For other similar calls we just assume s is non-NULL, and it is a programmer error if a NULL is provided.